html/components/_accordions.scss

```diff
@@ -35,18 +35,19 @@
 
 .sprk-c-Accordion__summary {
   align-items: center;
+  background-color: $sprk-accordion-summary-background-color;
+  border: $sprk-accordion-summary-border;
   color: $sprk-accordion-summary-color;
   display: flex;
   padding: $sprk-accordion-summary-padding;
-  text-decoration: none;
   text-align: $sprk-accordion-summary-text-align;
-  border: $sprk-accordion-summary-border;
-  background-color: $sprk-accordion-summary-background-color;
+  text-decoration: $sprk-accordion-summary-text-decoration;
+  width: 100%;
```

Comment:
Should this be a sass var?

Reply:
@bobwise the width 100% is okay to not be a var since if it’s not 100%, the touch target of the accordion item isn’t spanning full width and users could have issues on small screens. So we don’t want to make that configuration available.